profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.install

```diff
@@ -85,18 +85,96 @@ function tmgmt_poetry_schema() {
 }
 
 /**
+ * Implements hook_requirements().
+ */
+function tmgmt_poetry_requirements($phase) {
```

Comment:
Actually i think that the requirement is linked to the module, not to the feature. So it is strange to move a requirement outside its scope just to fix an problem in the mock.
I think it's better to change poetry mock and have poetry as a requirement and not a dependency, no?